source/includes/_iframe.md

```diff
@@ -0,0 +1,166 @@
+	
+	- `width: 100%;`: This way the iframe occupies all available width from parent
+	- `min-width: 320px;`: Below this mininum width, the iframe wouldn't display its contents correctly.
+	- `min-height: 560px`: Although this one it is just recommended, having this minimum height will make scrollbars disappear.
+
+
+## Communicating with the iframe events
+
+The iframe contains both part of the implementation the merchant would have on their side as well as some implementation that usually happens on the wallet side.
+
+Merchant would usually need to gather from the users what fiat amount and fiat currency they want to deposit and send it to the ON/RAMP api using the `create_ingress_invoice` endpoint to later redirect the users to the redirection url they got as a response from ON/RAMP. However, in this integration, the iframe will deal with gathering the information from the user and redirecting the user to the ON/RAMP wallet only if the payment was successful.
+
+Given that, merchant and the iframe need a way to communicate certain events and this is achieve by using the `window.postMessage()` [API](https://developer.mozilla.org/en-US/docs/Web/API/Window/postMessage).
+
+- Events sent from merchant:
+	
+	- `get_deposit_details`
+	- `create_ingress_invoice`
+
+- Events sent from iframe:
+
+	- `get_deposit_details`
+	- `create_ingress_invoice`
+
+Both of the events are going to be initiated by the iframe and the merchant will respond to them.
+
+### Example of integration
+
+On the right side, there is a full example of an implementation any merchant could use in order to integrate our iframe.
+
+Please, note the following: 
+
+- There are some functions that are not required and they have been added to show a complete integration that should just work. The only required implementation is the one about the `window.postMessage` and `window.addEventListener` APIs.
+
+- The example hardcode some values and that is marked with commments. In order to make this fully functional according tou your server and API, you will need to implement or adapt calls to store the deposits and make secure connections to the ON/RAMP API using your API key.
+
+> Example of integration
+
+```javascript
+const postMessage = (message) => {
+	const stringifiedMessage = JSON.stringify(message)
+	document.querySelector('#onramp-iframe').contentWindow.postMessage(stringifiedMessage, '*')
+}
+
+const createMessage = (eventName, data, error) => {
+	return {
+		event: eventName,
+		data,
+		error
+	}
+}
+
+const postData = async (url = '', data = {}) => {
+	try {
+		const response = await fetch(url, {
+			method: 'POST', // *GET, POST, PUT, DELETE, etc.
+			mode: 'cors', // no-cors, *cors, same-origin
+			cache: 'no-cache', // *default, no-cache, reload, force-cache, only-if-cached
+			credentials: 'same-origin', // include, *same-origin, omit
+			headers: {
+				'Content-Type': 'application/json'
+				// 'Content-Type': 'application/x-www-form-urlencoded',
```

Comment:
`x-www-form-urlencoded` won't ever be useful here, remove this comment?..

Reply:
Well, those were some comments to this postData function and explain how any of those parameters can be changed not for our requests to ON/RAMP but in general if somebody wants to reuse it.
Conceptually we could remove all comments here, but I thought it was good for any dev.